tests/manage/z_cluster/nodes/test_check_pod_status_after_two_nodes_shutdown_recovery.py

```diff
@@ -0,0 +1,110 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.framework.testlib import (
+    ManageTest, tier4c, ignore_leftovers
+)
+from tests.sanity_helpers import Sanity
+from ocs_ci.ocs.node import (
+    wait_for_nodes_status, get_typed_nodes
+)
+from ocs_ci.utility.retry import retry
+from ocs_ci.ocs.exceptions import CommandFailed, ResourceWrongStatusException
+from ocs_ci.ocs.resources.pod import (
+    wait_for_storage_pods, list_of_nodes_running_pods
+)
+
+log = logging.getLogger(__name__)
+
+
+@ignore_leftovers
+@tier4c
+class TestOCSWorkerNodeShutdown(ManageTest):
+    """
+    Test case validate both the MDS pods rbd and cephfs plugin Provisioner
+    pods and not running on same node post shutdown and recovery
+
+    """
+
+    @pytest.fixture(autouse=True)
+    def init_sanity(self):
+        """
+        Initialize Sanity instance
+
+        """
+
+        self.sanity_helpers = Sanity()
+
+    @pytest.mark.polarion_id("OCS-2315")
+    def test_node_after_shutdown_and_recovery_worker_node(self, nodes, node_restart_teardown):
```

Comment:
Please change the test function name according to the test case and break the line

Reply:
Done